infra/modules/service/main.tf

```diff
@@ -319,3 +319,28 @@ resource "aws_security_group" "app" {
     cidr_blocks = ["0.0.0.0/0"]
   }
 }
+
+###########################
+## Database Configuration ##
+###########################
+resource "aws_rds_cluster" "postgresql" {
+  cluster_identifier = "${var.service_name}"
+  engine = "aurora-postgresql"
+  engine_mode = "provisioned"
+  database_name = replace("${var.service_name}", "-", "_")
+  master_username = "app_usr"
+  master_password = "test12345"
```

Comment:
Even for a test, we probably shouldn't use a password hardcoded into the terraform

I have this example from my own terraform that uses secrets manager, and i believe similar is possible with parameter store (secure strings)
```terraform
  password                = data.aws_secretsmanager_secret_version.rds_admin_password.secret_string
```


```terraform
resource "random_password" "rds_admin_password_value"{
  length           = 24
  special          = true
  override_special = "_!%^"
}

resource "aws_secretsmanager_secret" "rds_admin_password" {
  name = "metadata/rds/admin-password"
}

resource "aws_secretsmanager_secret_version" "rds_admin_password_version" {
  secret_id = aws_secretsmanager_secret.rds_admin_password.id
  secret_string = random_password.rds_admin_password_value.result
}

```

Reply:
That's fairly similar to what we did on the demo.